### DISCUSSION TITLE
Problem with some special charactes importing CSV file.

### DISCUSSION THREAD

**Discussion by paconde** [AntlerVC/firetable#746] [Support / Q&A] [answered]
Hello, when importing a CSV file, I have problems with some special characters, such as "Ñ" or "º". Is there any solution?

**Comment by BeeBombshell** [comment]
Hi @paconde, I have created an issue discussing the problems you have been facing. Somebody will get back to you soon. To keep track, you can follow this [issue](https://github.com/rowyio/rowy/issues/748). 🙂

**Comment by htuerker** [comment, marked as answer]
Hi, @paconde!

I've investigated the problem, there isn't an actual problem related to the Rowy application itself. Everything works well on my machine.
![Screen Shot 2022-08-26 at 21 03 34](https://user-images.githubusercontent.com/46192266/186965261-b5e04e06-9796-4aa4-bb9a-2e754cc120f8.png)

![Screen Shot 2022-08-26 at 21 03 44](https://user-images.githubusercontent.com/46192266/186965277-323ffc1f-1543-46e3-94b4-9184ab3a14ca.png)

However, I'd like to share my opinions to help you find the problem.
- I am guessing that your machine already supports those special characters as they look nice in your text editor. But, � actually represents the "replacement char" which is used to indicate problems when a system is unable to render a character.
- First, let's be sure that it's not a client issue. Could you check if it appears the same on your firestore console
![Screen Shot 2022-08-26 at 21 07 24](https://user-images.githubusercontent.com/46192266/186965907-005b9c00-cb6d-41ed-b47e-2ca0b43f7c16.png)
- If so, how you export your CSV data might be the source of this problem. It should be Unicode(UTF-8) encoded. 
---
Feel free to tag me for your following questions, I'd like to give more help with more details.

**Comment by RajGM** [comment]
We can use icnov to translate non-english characters to printable ones.
https://stackoverflow.com/questions/54583483/how-to-read-special-characters-from-csv-files-in-javascript

**Comment by RajGM** [comment]
But I am getting dependencies error while trying to install icnov.

**Comment by RajGM** [comment]
`npm WARN config global --global, --local are deprecated. Use --location=global instead.
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: @craco/craco@6.4.3
npm ERR! Found: react-scripts@5.0.0
npm ERR! node_modules/react-scripts
npm ERR!   peer react-scripts@"^5" from craco-swc@0.5.1
npm ERR!   node_modules/craco-swc
npm ERR!     dev craco-swc@"^0.5.1" from the root project
npm ERR!   react-scripts@"^5.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react-scripts@"^4.0.0" from @craco/craco@6.4.3
npm ERR! node_modules/@craco/craco
npm ERR!   peer @craco/craco@"^6  ^7" from craco-swc@0.5.1
npm ERR!   node_modules/craco-swc
npm ERR!     dev craco-swc@"^0.5.1" from the root project
npm ERR!   dev @craco/craco@"^6.4.3" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: react-scripts@4.0.3
npm ERR! node_modules/react-scripts
npm ERR!   peer react-scripts@"^4.0.0" from @craco/craco@6.4.3
npm ERR!   node_modules/@craco/craco
npm ERR!     peer @craco/craco@"^6  ^7" from craco-swc@0.5.1
npm ERR!     node_modules/craco-swc
npm ERR!       dev craco-swc@"^0.5.1" from the root project
npm ERR!     dev @craco/craco@"^6.4.3" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\RajGM\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\RajGM\AppData\Local\npm-cache_logs\2022-09-18T17_43_50_307Z-debug-0.log`
